wordpress-mobile/WordPress-iOS
# Removes the what's new popup.

Removes the what's new popup in order to resolve [this issue](https://github.com/wordpress-mobile/WordPress-iOS/issues/4573).

The reason why I'm removing the what's new popup code in its entirety is because I don't feel it was programmed to be reusable enough to make it worth keeping.

**Test 1:**
1. Launch the app for the first time.
2. Make sure the app is usable without the popup coming up at all.

**Test 2:**
1. Launch the app a second time.
2. Make sure the app is usable without the popup coming up at all.
